cmd/prometheus/main.go

```diff
@@ -278,6 +278,11 @@ func main() {
 		os.Exit(2)
 	}
 
+	if _, err := config.LoadFile(cfg.configFile); err != nil {
```

Comment:
A comment that we're doing this so we'll fail early would be useful.

Reply:
Done. I hope that is satisfactory.